packages/ui/react-ui-editor/src/components/EditorMenuProvider/EditorMenuProvider.tsx

```
@@ -156,7 +152,7 @@ type MenuProps = {
 const Menu = ({ groups, currentItem, onSelect }: MenuProps) => {
   const { tx } = useThemeContext();
   return (
-    <ul>
+    <ul role='menu'>
```

Biome (2.4.4)

[error] The HTML element ul is non-interactive and should not have an interactive role.

(lint/a11y/noNoninteractiveElementToInteractiveRole)